src/utils.ts

```diff
@@ -366,6 +366,10 @@ export const isAwsService = (host, responseData = undefined): boolean => {
   );
 };
 
+const checkInput = (input, words) => {
+  return words.some((word) => new RegExp(word, 'i').test(input));
```

Comment:
define the words as regexes from the first place. You don't need to compile it into regexes every time...